File: DiceGame/DiceGame_csharp/DiceGame/DiceGame.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DiceGame
{
	class DiceGame
	{
		Dice m_d1;
		Dice m_d2;

		public DiceGame()
		{
			m_d1 = new Dice();
			m_d2 = new Dice();
		}

		public bool Play()
		{
			m_d1.Roll();
			m_d2.Roll();
			return m_d1.GetValue() + m_d2.GetValue() == 7;
		}
	}
}

File: DiceGame/DiceGame_kotlin/DiceGame.kt
class DiceGame {
    val WIN_CONDITION: Int = 7
    var dice1: Dice = Dice()
    var dice2: Dice = Dice()

    fun play(): Boolean {
        dice1.roll()
        dice2.roll()

        return dice1.getValue() + dice2.getValue() == WIN_CONDITION
    }
}